smart_card_connector_app/src/managed_storage_schema.json

```diff
@@ -3,7 +3,7 @@
   "properties": {
     "expose_chrome_smart_card_api": {
       "title": "Handle requests from chrome.smartCardProviderPrivate API.",
-      "description": "If set to true, the app will subscribe to chrome.smartCardProviderPrivate API events and handle the requests. This is needed for Web Smart Card API.",
+      "description": "DEPRECATED. If set to true, the app will subscribe to chrome.smartCardProviderPrivate API events and handle the requests. This is needed for Web Smart Card API.",
```

Comment:
nit: Let's also edit that old description to say that this functionality is now on by default.

Reply:
done